README.md

```diff
@@ -1,20 +1,14 @@
 [![Main and pull request checks](https://github.com/Aiven-Open/commons-for-apache-kafka-connect/actions/workflows/main_push_workflow.yml/badge.svg)](https://github.com/Aiven-Open/commons-for-apache-kafka-connect/actions/workflows/main_push_workflow.yml)
 
-# Aiven's Common Module for Apache Kafka® connectors
+# Aiven's Apache Kafka Connectors
 
-Shared common functionality among Aiven's connectors for Apache Kafka:
-- [Aiven GCS Connector](https://github.com/aiven-open/gcs-connector-for-apache-kafka)
-- [Aiven S3 Connector](https://github.com/aiven-open/s3-connector-for-apache-kafka)
-
-# Usage
-
-When installing this library on Kafka Connect, use a specific plugin path, and **avoid placing them on the same path as the Kafka Connect binaries**, as some libraries may have conflicting versions.
```

Comment:
Could be better to mention about installation of these libraries are described in the readme files under specific connect folders

Reply:
And is commons module required to install gcs or s3 ? If so, we can mention that as well in this read me

Reply:
Commons is a dependency for both connectors.